[Add-ins] (Visual Studio mentions) Updates VS 17 instructions to VS 19 (#1372)
OfficeDev/office-js-docs-pr: docs/publish/host-an-office-add-in-on-microsoft-azure.md

1. Start Visual Studio as an administrator.

2. Choose **Create a new project**.

3. Using the search box, enter **add-in**.

4. Choose **Word Web Add-in** as the project type, and then choose **Next** to accept the default settings.

> Visual Studio creates a basic Word add-in that you'll be able to publish as-is, without making any changes to its web project. In order to make an add-in of a different Office host type, such as Excel, repeat the steps and choose another project type with your desired Office host.

Reviewer: ```suggestion
Visual Studio creates a basic Word add-in that you'll be able to publish as-is, without making any changes to its web project. To make an add-in for a different Office host type, such as Excel, repeat the steps and choose a project type with your desired Office host.
```

3. In the XML manifest file, find and replace all instances of "~remoteAppUrl" with the root URL of the add-in web app on Azure. This is the URL that you copied earlier after you published the add-in web app to Azure (for example: https://YourDomain.azurewebsites.net). 

4. Choose **File** and then choose **Save All**. Next, Copy the add-in XML manifest file (for example, WordWebAddIn.xml).

> 5. Using the separate program **File Explorer**, browse to the network file share that you created in [Step 1: Create a shared folder](../publish/host-an-office-add-in-on-microsoft-azure.md#step-1-create-a-shared-folder-to-host-your-add-in-xml-manifest-file) and paste the manifest file into the folder.

Reviewer: ```suggestion
5. Using the **File Explorer** program, browse to the network file share that you created in [Step 1: Create a shared folder](../publish/host-an-office-add-in-on-microsoft-azure.md#step-1-create-a-shared-folder-to-host-your-add-in-xml-manifest-file) and paste the manifest file into the folder.
```
